packages/app/styles.css

```diff
@@ -65,6 +65,15 @@ body {
     margin-bottom: 2rem;
 }
 
+.form-group label:has(input[type='checkbox']) {
```

Comment:
This adjusts the spacing between the RSJF checkbox and its label. That had been bothering me for too long, haha.

Reply:
😂 Thank you this has always annoyed me too